decentraland/components/presentational/MyAssetsList.js

```diff
@@ -11,27 +11,35 @@ import LargeText from "@presentational/LargeText";
 // https://medium.com/groww-engineering/stateless-component-vs-pure-component-d2af88a1200b
 export default class MyAssetsList extends React.PureComponent {
   render() {
-    const { myAssetsList, renderItem } = this.props;
-    const { length: listAmount } = myAssetsList;
+    const { myAssets, userActions, renderItem } = this.props;
+    const { length: listAmount } = myAssets;
```

Comment:
I forget, is length explicitly defined or is this a newer ECMAScript feature?

Reply:
(Not gating merging this PR)

Reply:
Every array has the length property

Reply:
Ahh yeah of course, this is basically the same as doing `myAssets.length`, and I remember that that's possible haha

Reply:
Resolving this